galasa-managers-parent/galasa-managers-visualisation-parent/dev.galasa.elasticlog.manager/src/main/java/dev/galasa/elasticlog/internal/properties/ElasticLogEndpoint.java

```diff
@@ -0,0 +1,40 @@
+package dev.galasa.elasticlog.internal.properties;
```

Comment:
What about credentials.   the elastic server will need to be secured to prevent naughty UPs from sending false data